src/tests/APICore.spec.ts

```diff
@@ -188,37 +188,6 @@ describe('APICore', () => {
             });
         });
 
-        /**
-         * @deprecated Will be removed in version 45
-         */
-        describe('getFile', () => {
```

Comment:
Shouldn't we have a test to test the blob with the get method?

Reply:
We have multiple tests across the resources that do that, for example:
- https://github.com/coveo/platform-client/blob/dc3671cf2ff185cf2c6b69d4d5a558b2cff23477/src/resources/ResourceSnapshots/tests/ResourceSnapshots.spec.ts#L142
- https://github.com/coveo/platform-client/blob/dc3671cf2ff185cf2c6b69d4d5a558b2cff23477/src/resources/Pipelines/Statements/tests/Statements.spec.ts#L46
- https://github.com/coveo/platform-client/blob/b9cd9d13e71c6d1394a8555abe5dc9ba912653f9/src/resources/Search/test/Search.spec.ts#L158

In the APICore we have the following test:

https://github.com/coveo/platform-client/blob/3c937c90ddff6559d0a17f7019adf8ebc778ab56/src/tests/APICore.spec.ts#L450

Is that enough for you? For me it is.

Reply:
I'd hate to think that it's the resource tests that are testing the APICore file for generic functionality.

I think it would be a good idea to have a test in APICore with a blob, as we're deleting the getFile method that used to do this job.

On the other hand, will the test you show in APICore really be any different if we test with a blob? If not, then maybe the test is useless?

Reply:
As explained in DM, repeating the test but for the string `blob` would not add any coverage. The logic to handle blobs is on the response handlers side and its already tested here https://github.com/coveo/platform-client/blob/589938247917fb4f5d65680f0514bf18b6002ebd/src/handlers/tests/ResponseHandlers.spec.ts#L28